core/test/unit/lib/mobiledoc/converters/mobiledoc-converter_spec.js

```diff
@@ -1,34 +1,111 @@
-var should = require('should'), // jshint ignore:line
-    converter = require('../../../../../server/lib/mobiledoc/converters/mobiledoc-converter');
-
-describe('Convert mobiledoc to HTML ', function () {
-    var mobiledoc = {
-        version: '0.3.1',
-        atoms: [],
-        cards: [
-            ['markdown',
-                {
-                    pos: 'top',
-                    card_name: 'markdown',
-                    markdown: '#heading\n\n- list one\n- list two\n- list three'
-                }
-            ],
-            ['html', {
-                pos: 'top',
-                card_name: 'html',
-                html: '<p>HTML CARD</p>'
-            }]
-        ],
-        markups: [],
-        sections: [
-            [1, 'p', [
-                [0, [], 0, 'test']
-            ]],
-            [10, 0],
-            [10, 1]
-        ]
-    };
-    it('Converts a mobiledoc to HTML', function () {
-        converter.render(mobiledoc).should.match('<p>test</p><div class="kg-card-markdown"><h1 id="heading">heading</h1>\n<ul>\n<li>list one</li>\n<li>list two</li>\n<li>list three</li>\n</ul>\n</div><div class="kg-card-html"><p>HTML CARD</p></div>');
+'use strict';
+
+const should = require('should'); // jshint ignore:line
+const converter = require('../../../../../server/lib/mobiledoc/converters/mobiledoc-converter');
+
+describe('Mobiledoc converter', function () {
+    // version 1 === Ghost 1.0 markdown-only renderer
+    describe('version 1', function () {
+        it('renders correctly', function () {
+            let mobiledoc = {
+                version: '0.3.1',
```

Comment:
We already have a helper fn in our DataGenerator, `markdownToMobiledoc`.
It would be helpful to keep the structure of mobiledoc in one place.

See `test/utils/fixtures/DataGenerator`.

Reply:
That util is for converting a markdown string to markdown-only mobiledoc, it's not relevant in this test case at all.

Reply:
Ah sorry, for some reason I thought this was related to a different test 😖 I think in that case that explicitness is preferable? Not a fan of having indirection in tests that are directly related to testing input and output